extensions/container-image/container-image-docker-common/deployment/src/main/java/io/quarkus/container/image/docker/common/deployment/RedHatOpenJDKRuntimeBaseProvider.java

```diff
@@ -22,7 +21,7 @@ public Optional<DockerFileBaseInformation> determine(Path dockerFile) {
             if (fromOpt.isPresent()) {
                 String fromLine = fromOpt.get();
                 String baseImage = fromLine.substring(4).trim();
-                Pattern pattern = Pattern.compile(".*ubi8/openjdk-(\\w+)-runtime.*");
+                Pattern pattern = Pattern.compile(".*ubi[89]/openjdk-(\\w+)-runtime.*");
```

Comment:
These kinds of behavior changes should be separate commits

Reply:
Sure. I can revert this and move to a separate commit.

Reply:
Done